lib/interfaces/src/CCUEthernetInterface.cpp

```diff
@@ -9,10 +9,12 @@
 
 //take the values from the protobuf message and update CCUInput_s 
 
-void ethernetInterface::recieve_pb_msg_acu_all_data(hytech_msgs_ACUAllData_s &msg_in, CCUInput_s &inputValues){
-
-    inputValues.max_cell_voltage = *std::max_element(msg_in.voltages, msg_in.voltages + msg_in.voltages_count);
-
-    inputValues.max_cell_temperature = *std::max_element(msg_in.cell_temperatures, msg_in.cell_temperatures + msg_in.cell_temperatures_count);
-
+void ethernetInterface::recieve_pb_msg_acu_all_data(const hytech_msgs_ACUAllData_s &msg_in, ACUAllData_s &acu_all_data){
+
+    for (uint32_t i = 0; i < msg_in.voltages_count; ++i){
+        acu_all_data.voltages[i] = msg_in.voltages[i];
+    }
+    for (uint32_t i = 0; i < msg_in.cell_temperatures_count; ++i){
+        acu_all_data.cell_temperatures[i] = msg_in.cell_temperatures[i];
+    }
```

Comment:
the namespace name should be something CCUEthernetInterface to be consistent with vcf/vcr